src/library/ksModal/ksModal.scss

```
@@ -0,0 +1,31 @@
+@import '../../styles/variables.scss';
```

```suggestion
@import '../../styles/variables';
```

This file is now a scss partial that will only contain css variables, should be imported like so ☝️